speech/cloud-client/src/main/java/com/example/speech/Recognize.java

```diff
@@ -186,6 +189,11 @@ public static void asyncRecognizeFile(String fileName) throws Exception, IOExcep
       List<SpeechRecognitionAlternative> alternatives = result.getAlternativesList();
       for (SpeechRecognitionAlternative alternative: alternatives) {
         System.out.printf("Transcription: %s%n", alternative.getTranscript());
+        for (WordInfo wordInfo: alternative.getWordsList()) {
```

Comment:
Done. I added the test. I also fixed the test failure in AsyncRecognizeGcs test, so all the tests should pass now